typings/index.d.ts

```diff
@@ -1977,14 +1977,26 @@ declare module 'discord.js' {
     constructor(channel: TextChannel | DMChannel, iterable?: Iterable<any>);
     public channel: TextBasedChannelFields;
     public cache: Collection<Snowflake, Message>;
+    public crosspost(message: MessageResolvable): Promise<Message>;
+    public delete(message: MessageResolvable): Promise<void>;
+    public edit(
+      ...args:
+        | [
+            message: MessageResolvable,
+            content: APIMessageContentResolvable | MessageEditOptions | MessageEmbed | APIMessage,
+          ]
+        | [message: MessageResolvable, content: StringResolvable, options: MessageEditOptions | MessageEmbed]
+    ): Promise<Message>;
```

Comment:
Why not make those 2 separate overloads, instead of one using rest and tuple syntax?

Reply:
Honestly, this came from 7bf12fb and I don't at all remember why